src/fragment-components/tiles/a-clickable-tile.tsx

```diff
@@ -113,6 +113,7 @@ export const componentInfo: ComponentInfo = {
 	codeUI: AClickableTileCodeUI,
 	keywords: ['tile', 'clickable', 'card'],
 	name: 'Clickable tile',
+	type: 'clickabletile',
```

Comment:
```suggestion
	type: 'clickable-tile',
```

Reply:
Thought we agreed to use a dashed naming convention for types? Should also apply to all of the ones below except `tile`